docs/DeadmanAlert.md

```diff
@@ -0,0 +1,60 @@
+### Deadman Alert
+
+The `deadman` function is used in the context of a task to alert when a group stops reporting data.
+For example, say you have the following task definition:
+
+```
+option task = {
+    ...
+    every: 15m,
+    ...
+}
+from(bucket: "telegraf/autogen")
+    |> range(start: -1h)
+    |> filter(fn: (r) => r._measurement == "cpu")
+```
+
+In order to be alerted when the above task stops reporting data for a group, you would define a deadman alert using the `deadman` function like so:
+```
+option task = {
+    ...
+    every: 15m,
+    ...
+}
+from(bucket: "telegraf/autogen")
+    |> range(start: -1h)
+    |> filter(fn: (r) => r._measurement == "cpu")
+    |> deadman(d: task.every)
+    |> alert(crit: (r) => r.dead)
+```
```

Comment:
@jlapacik I know I came late and this got merged, but I have a concern about this.

If you specify `deadman` with `d` equal to `task.every` you will never get what you want, because every series will  contain the data for the specified time interval. If  the series doesn't contain data it won't be reported and you wouldn't be able to trigger an alert.

You should always  specify a `d < task.every` in order for the query to work.

Is this concern vane?

Reply:
We should state out clealry (at least in the doc) that, if using `deadman` within a task, it should be that `task.every > d`. So, you should always make your task sample data for an interval greater than the deadman duration.

I know this is only an example, just to make sure we are on the same page :).

Reply:
@affo it's a valid concern however the above alert is valid because the data sampled is for `-1h` and `d < 1h`. `task.every` is just how frequently the task runs. Every time the task runs it reads the last `1h` worth of data.

Reply:
@jlapacik Absolutely, I haven't noticed that. Thank you.